.github/workflows/cts.yml

```diff
@@ -99,7 +99,11 @@ jobs:
         if: matrix.os == 'ubuntu-24.04'
         uses: ./.github/actions/install-mesa
 
-      - name: run CTS
+      - name: Test cts_runner
+        shell: bash
+        run: cargo --locked xtask test --llvm-cov -p cts_runner
```

Comment:
I don't think it's particularly interesting or useful for these to contribute to coverage, but it seems like building with `--llvm-cov` maximizes build reuse between this and the next step.